Sources/SmileID/Classes/SmileID.swift

```diff
@@ -6,7 +6,7 @@ import UIKit
 public class SmileID {
     /// The default value for `timeoutIntervalForRequest` for URLSession default configuration.
     public static let defaultRequestTimeout: TimeInterval = 60
-    public static let version = "10.5.2"
+    public static let version = "10.99.99"
```

Comment:
Looks like this was added for testing purposes?

Reply:
Yes, we need to revert this to the sdk version under which we release.